### ISSUE TITLE
bug: webapp billing period description

### ISSUE THREAD

**Issue by ubinatus** [triggerdotdev/trigger.dev#1445]
### Describe the bug

Minor bug on the billing period message that displays an invalid period range:

<img width="1176" alt="image" src="https://github.com/user-attachments/assets/d58769ee-6168-49ab-9252-0dc2353ff896">

Should be from Oct1 to Oct 31 (or Nov 1) but not Nov 30. Might be because today is the last day of the month?

Current locale: GMT-5

https://github.com/triggerdotdev/trigger.dev/blob/da08e5015f297d21619aa45190fafd0b0ca460dd/apps/webapp/app/routes/_app.orgs.%24organizationSlug.v3.billing/route.tsx#L44-L56

**Comment by matt-aitken** [comment]
Yes I'm sure you're correct on why this is happening. The UTC dates are correct but then we display using our `<DateTime>` React component which renders in your local browser timezone by default…

**Comment by matt-aitken** [comment]
Actually ignore my last comment. That wouldn't explain why it's coming back as November 30… 😕  This is strange because the code you've included runs on our server which is UTC. 

The way to fix this is with the debugger on and setting your timezone

**Comment by matt-aitken** [comment]
The billing page has been rebuilt since this report. Closing as outdated. Please open a new issue if the period description is still incorrect.
